crates/typst-library/src/foundations/float.rs

```diff
@@ -110,7 +110,8 @@ impl f64 {
         f64::signum(self)
     }
 
-    /// Converts bytes to a float.
+    /// Interprets bytes as a float in [IEEE 754](https://en.wikipedia.org/wiki/IEEE_754)
```

Comment:
At the moment, the first line of the docs shouldn't have a link because various places in the docs generation and IDE tooling expects the first line to be plain text.

Reply:
I moved the link to the docs of the relevant argument; does that resolve the problem?

Reply:
Yes!